arrows/ocv/tests/test_image.cxx

```diff
@@ -344,17 +344,56 @@ TYPED_TEST(image_conversion, vital_to_ocv_interleaved)
   run_vital_conversion_tests( img );
 }
 
+// ----------------------------------------------------------------------------
+TEST(image, bgr_to_rgb)
+{
+  // Create vital images and convert to and from cv::Mat
+  // (note: different code paths are taken depending on whether the image
+  // is natively created as OpenCV or vital, so we need to test both ways)
+  kwiver::vital::image_of<unsigned char> img{ 200, 300, 3 };
+  populate_vital_image<unsigned char>( img );
+  cv::Mat ocv_img =  ocv::image_container::vital_to_ocv(img, ocv::image_container::BGR_COLOR);
+  {
+    int const num_c = ocv_img.channels();
+    for( int j = 0; j < ocv_img.rows; ++j )
```

Comment:
What is the purpose of testing each channel separately? (Rather than testing all three in only one loop?)